content/W2016/W16-media-stephanie.md

```diff
@@ -0,0 +1,20 @@
+Title: Stephanie Morillo: “Circuit Breaker”
+Date: 2016-03-22
+Category: Media
+Tags: talks, #WocinTech, technical writing
+Slug: stephanie-morillo-video-2016
+Author: Fatema Boxwala
+Summary: The amazing Stephanie Morillo gives a talk to UW students about
```

Comment:
wow I wonder how many times I'm going to make this mistake before I get it right